### PR TITLE
Add acknowledgements for deep-review authors

### LINKED ISSUE #50: Ping deep-review contributors to determine if they want to be acknowledged
We will `@mention` the contributors to each here. Those who respond by a defined date will be included with the acknowledgements.

### PR DESCRIPTION
**Pull request by cgreene**
This contains everyone who approved up to https://github.com/greenelab/meta-review/issues/50#issuecomment-397397736

I have also noted in that issue with a thumbs up when the person is acknowledged.

_(agitter edit: closes #50)_

### CODE REVIEW

**Comment by agitter** [comment]
Do you want to merge this or keep it open as more people respond?

**Comment by cgreene** [comment]
I was thinking we would just keep it open as more people respond.

**Comment by agitter** [comment]
This won't introduce conflicts so 👍

**Comment by cgreene** [comment]
Now has all up to https://github.com/greenelab/meta-review/issues/50#issuecomment-397686932

**Comment by cgreene** [comment]
Now up to date through https://github.com/greenelab/meta-review/issues/50#issuecomment-402763721

File: content/99.back-matter.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## Acknowledgements
 
-`TODO: deep review authors for support in testing this process`
+We would like to thank the authors of the Deep Review who helped us test this process.
```
Comment (agitter) [inline content/99.back-matter.md]: How about `test collaborative writing with Manubot` instead of `test this process`?

**Comment by agitter** [comment]
I reviewed #50 again and added the last missing deep review author who responded there.  I made the phrasing change that I suggested above.  I also updated everyone's names to match what they provided for deep review.

Of the 36 deep review authors, 28 responded and are acknowledged, 3 are meta review authors, and the following 5 have not yet responded:
- Johnny Israeli
- Marwin H. S. Segler
- Simina M. Boca
- S. Joshua Swamidass
- Austin Huang

I'm using full names instead of usernames so as not to bother authors who did not respond in #50.

@dhimmel can you please review once more so we can merge this?

**Review by dhimmel — APPROVED** [review]
Great! Thanks @agitter for doing the work here, as with this many contributors, it's a considerable amount of work and record keeping!

**Comment by agitter** [comment]
Merged.  Thanks @cgreene for compiling the names.

**Pull request merged**